packages/video_player/video_player_platform_interface/lib/messages.dart

```diff
@@ -1,25 +1,26 @@
-// Autogenerated from Pigeon (v0.1.12), do not edit directly.
+// Autogenerated from Pigeon (v0.1.17), do not edit directly.
 // See also: https://pub.dev/packages/pigeon
 // ignore_for_file: public_member_api_docs, non_constant_identifier_names, avoid_as, unused_import
 // @dart = 2.12
```

Comment:
note that pigeon outputs (and has always outputted ever since this feature was added) "2.10" here. I changed it manually to "2.12" to match what was here before, but that seems dubious since this file is autogenned.

Reply:
yep. That was me.  I sent https://github.com/flutter/packages/pull/252, but I haven't been able to make the test pass in the CI. (It does locally)